client/chat/channel-info-card.tsx

```diff
@@ -166,6 +171,12 @@ export function ConnectedChannelInfoCard({
     dispatch(getBatchChannelInfo(channelId))
   }, [dispatch, channelId])
 
+  useEffect(() => {
+    if (bannerId) {
+      dispatch(getBatchChannelBanner(bannerId))
```

Comment:
Can we not assume that `getBatchChannelInfo` already has this info? Seems like we're just making 2 requests for the same thing here